source/form.html

```diff
@@ -0,0 +1,316 @@
+              <ul class="achievement-list">
+                <li class="achievement-item">
+                  <input
+                    class="hidden"
+                    type="checkbox"
+                    name="selfie"
+                    id="selfie"
+                    checked
+                  />
+                  <label for="selfie">Сделал селфи с акулой</label>
+                </li>
+                <li class="achievement-item">
+                  <input
+                    class="hidden"
+                    type="checkbox"
+                    name="beach"
+                    id="beach"
+                    checked
+                  />
+                  <label for="beach">Обгорел на пляже</label>
+                </li>
+                <li class="achievement-item">
+                  <input
+                    class="hidden"
+                    type="checkbox"
+                    name="chuck"
+                    id="chuck"
+                  />
+                  <label for="chuck">Видел Чака Норриса</label>
+                </li>
+                <li class="achievement-item">
+                  <input
+                    class="hidden"
+                    type="checkbox"
+                    name="souvenir"
+                    id="souvenir"
+                  />
+                  <label for="souvenir">Накупил сувениров</label>
+                </li>
+                <li class="achievement-item">
+                  <input
+                    class="hidden"
+                    type="checkbox"
+                    name="tower"
+                    id="tower"
+                  />
+                  <label for="tower">Удержал башню</label>
+                </li>
+                <li class="achievement-item">
+                  <input
+                    class="hidden"
+                    type="checkbox"
+                    name="hotel"
+                    id="hotel"
+                    checked
+                  />
+                  <label for="hotel">Разгромил отель</label>
+                </li>
+              </ul>
+            </fieldset>
+
+            <!--  Phone and email -->
+
+            <fieldset>
+              <legend>контактная информация</legend>
+              <label for="phone">Номер телефона</label>
+              <input
+                type="tel"
+                id="phone"
+                name="phone"
+                placeholder="Номер, пожалуйста"
+              />
+              <label for="email">Адрес почты</label>
+              <input
+                type="email"
+                id="email"
+                name="email"
+                placeholder="Введите почту *"
+                required
+              />
+            </fieldset>
+
+            <!--  App -->
+
+            <fieldset>
+              <legend>С каким приложением путешествовали?</legend>
+              <ul class="application-list">
+                <li class="application-item">
```

Comment:
```suggestion
                <li class="application-list__item">
```
БЭМ элемент блока `application-list`.